src/components/course/CourseRunCard.jsx

```diff
@@ -102,8 +107,18 @@ const CourseRunCard = ({
   } = useSubsidyDataForCourse();
 
   const sku = useMemo(
-    () => findHighestLevelSeatSku(seats),
-    [seats],
+    () => {
+      const skuForSeats = findHighestLevelSeatSku(seats);
```

Comment:
Executive Education (2U) courses do have a seat, but with a mode of `unpaid-executive-education`, which is not explicitly handled by `findHighestLevelSeatSku`, so there is (correctly) no SKU determined based on the seats.

Instead, the product sku for Executive Education (2U) content is being stored on the course entitlement record along with the price. We believe the entitlement sku must be used for Executive Education (2U) courses, not the unpaid seat SKU.

So, the logic here is to fallback to any existing course entitlement SKU, assuming a seat SKU isn't returned already. This approach should be generic enough to avoid having to have a conditional specifically for "course_type: executive-education-2u".